util/useremails.py

```diff
@@ -67,7 +69,15 @@ def send_email(recipient, subject, template_file, parameters, action=None):
     msg.body = plain
 
     try:
-        mail.send(msg)
+        if features.FIPS:
+            assert app.config[
+                "MAIL_USE_TLS"
+            ], "MAIL_USE_TLS must be enabled to use SMTP in FIPS mode."
+            with mock.patch("smtplib.SMTP.login", login_fips_safe):
```

Comment:
Same question as above: is `smtplib.SMTP.login = login_fips_safe` or is `mock` needed?

Reply:
mock is needed since we want to localize the change for only the block below (using the with keyword).